packages/shared/src/models/IModelInfo.ts

```diff
@@ -29,4 +29,9 @@ export interface ModelInfo {
   file?: LocalModelInfo;
   state?: 'deleting';
   memory: number;
+  /**
+   * the chat format to use
+   * @example llama-2
+   */
+  chatformat?: 'openchat' | 'llama-2' | string
```

Comment:
the `| string` is making the other one useless

Reply:
The `| string` is here to allow arbritrary value, while showing the possible existing supported values

I took inspiration from the OpenAI lib

![image](https://github.com/containers/podman-desktop-extension-ai-lab/assets/42176370/56c47031-d1ee-4d61-b912-ea8023907b38)